packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -110,7 +115,7 @@ export class Alias extends FunctionBase {
     return super.metric(metricName, {
       dimensions: {
         FunctionName: this.underlyingLambda.functionName,
-        Resource: this.functionArn
+        Resource: `${this.underlyingLambda.functionArn}:${this.aliasName}`
```

Comment:
Provide some reasoning for why we are doing it like this here. Reference an issue?